lib/cc/analyzer/container.rb

```diff
@@ -67,6 +67,13 @@ def run(options = [])
         end
       end
 
+      def stop
+        @on_output = ->(*) { }
+        Process.kill("TERM", @pid)
+      rescue Errno::ESRCH
+        Analyzer.logger.info("Tried to send TERM to process with PID #{@pid}, but not found.")
```

Comment:
I was getting a Code Climate issue for leaving this empty. I think a statsd increment sounds good.